ci/travis/before_install-linux.sh

```diff
@@ -43,20 +43,13 @@ python -c 'import pip; print "pip version=", pip.__version__'
 python -c 'import setuptools; print "setuptools version=", setuptools.__version__'
 python -c 'import wheel; print "wheel version=", wheel.__version__'
 
-pip uninstall numpy --yes
-
 # Fetch nupic.core build
-export NUPIC_CORE_COMMITISH=`python -c "execfile('.nupic_modules'); print NUPIC_CORE_COMMITISH"`
-echo "Downloading nupic.core build: https://s3-us-west-2.amazonaws.com/artifacts.numenta.org/numenta/nupic.core/nupic_core-${NUPIC_CORE_COMMITISH}-linux64.tar.gz"
-curl -O "https://s3-us-west-2.amazonaws.com/artifacts.numenta.org/numenta/nupic.core/nupic_core-${NUPIC_CORE_COMMITISH}-linux64.tar.gz"
-tar xzf "nupic_core-${NUPIC_CORE_COMMITISH}-linux64.tar.gz"
-
-ls home/travis/build/numenta/nupic.core/bindings/py/dist/wheels
-
-# Install nupic.bindings and dependencies from wheels
-pip install --no-index --find-links=home/travis/build/numenta/nupic.core/bindings/py/dist/wheels nupic.bindings
-
-# Workaround for multiprocessing.Queue SemLock error from run_opf_bechmarks_test.
-# See: https://github.com/travis-ci/travis-cookbooks/issues/155
-# Commented out to test to see if it works witout it in container mode.
-# sudo rm -rf /dev/shm && sudo ln -s /run/shm /dev/shm
+while read line; do
+    if [[ $line == nupic.bindings* ]];
+    then
+        IFS='=' read -ra ADDR <<< "$line"
+        NUPIC_BINDINGS_VERSION="${ADDR[2]}"
+    fi
+done <requirements.txt
+
+pip install https://s3-us-west-2.amazonaws.com/artifacts.numenta.org/numenta/nupic.core/releases/nupic.bindings/nupic.bindings-${NUPIC_BINDINGS_VERSION}-cp27-none-linux_x86_64.whl
```

Comment:
Still have to get the linux binary from S3 until we have manylinux binaries in pypi.